bundles/org.openhab.binding.emotiva/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -260,8 +260,8 @@
 		<description>Speaker Preset Name</description>
 		<state>
 			<options>
-				<option value="preset1">Preset 1</option>
-				<option value="preset2">Preset 2</option>
+				<option value="preset1">@text/channel-type.emotiva.speaker-preset.state.option.preset-1</option>
```

Comment:
It should not be needed to add a custom key as these options are translated by default. Or what am I missing?

Reply:
Did not work before, so I looked into the Miele binding to find this solution, but I was actually missing the "state" part for every line, so now it works as intended with the short snart in the value.